root/static/scripts/relationship-editor/components/RelationshipDialogContent.js

```diff
@@ -201,6 +205,57 @@ export function createInitialState(props: PropsT): RelationshipDialogStateT {
   };
 }
 
+/*
+ * Possibly updates newState.backward based on the source and target
+ * entities and the link type. The link direction is updated only in
+ * cases where the correct value can be inferred, e.g. "member of"
+ * relationships between person and group artist entities. See MBS-2604.
+ */
+function inferLinkDirection(newState: {...RelationshipDialogStateT}): void {
```

Comment:
I wasn't sure if this function can live here or if it should be in a different file. Conceptually, it doesn't feel very different from the existing `updateDialogStateForTargetTypeChange` function that's defined after it.

Reply:
It is a good place for now.